app/Device.php

```diff
@@ -262,4 +257,41 @@ public function data()
     {
         return $this->hasManyThrough('App\SensorData', 'App\Sensor');
     }
+    
+    /**
+     * Is the device ready for a command
+     *
+     * @return boolean
+     */
+    public function isReadyForCommand()
+    {
+        $isReady = false;
+        
+        if ($this->cover_status == 'open' || $this->cover_status == 'closed' || $this->cover_status == 'locked')
+            $isReady = true;
+        
+        return $isReady;
+    }
+    
+    /**
+     * Is the device schedule during open hours
+     *
+     * @return boolean
+     */
+    public function isDuringScheduleOpen()
+    {
+        $isOpenHours = false;
+    
+        $timezone = Auth::user()->timezone;
+        //Get the open, close, and current time in the users timezone
+        $open_time = new Carbon($this->open_time, $timezone);
+        $close_time = new Carbon($this->close_time, $timezone);
+        $time_now = Carbon::now($timezone);
+    
+        //Check if the current time is during the open schedule or not
+        if ($time_now->gt($open_time) && $time_now->lt($close_time))
+            $isOpenHours = true;
+        
+        return $isOpenHours;
```

Comment:
Just return the boolean expression, there are no other conditionals requiring the use of the boolean variable.